sql/testdata/datetime

```diff
@@ -42,6 +42,44 @@ statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\)
 INSERT INTO t VALUES
   ('2015-08-29 20:34:45.34567-07:00'::timestamp, '2015-08-31'::date, '35h2s'::interval)
 
+statement ok
+SET TIME ZONE 'EUROPE/ROME'
+
+statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\) violates unique constraint "primary"
+INSERT INTO t VALUES
+  ('2015-08-30 04:34:45.34567 CET'::timestamp, '2015-08-31'::date, '35h2s'::interval)
+
+statement ok
+SET TIME ZONE 'America/New_York'
+
+statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\) violates unique constraint "primary"
+INSERT INTO t VALUES
+  ('2015-08-29 22:34:45.34567 EST'::timestamp, '2015-08-31'::date, '35h2s'::interval)
+
+statement error cannot find foobar.*
+SET TIME ZONE 'foobar'
+
+statement ok
+SET TIME ZONE INTERVAL '-7h'
+
+statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\) violates unique constraint "primary"
+INSERT INTO t VALUES
+  ('2015-08-29 20:34:45.34567'::timestamp, '2015-08-31'::date, '35h2s'::interval)
+
+statement error time: invalid duration foobar
+SET TIME ZONE INTERVAL 'foobar'
+
+# No tests for LOCAL and DEFAULT.
```

Comment:
I think we should make `LOCAL` and `DEFAULT` synonyms for UTC and then we can test them.